apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerConnection.cs

```diff
@@ -116,6 +119,27 @@ public override ConnectionState State
         /// </summary>
         public QueryOptions QueryOptions { get; set; }
 
+        /// <summary>
+        /// Request CommitStats for all read/write transactions throughout the
+        /// lifetime of the connection.
+        /// </summary>
+        public bool ReturnCommitStats { get; set; }
```

Comment:
To give some additional context that I may not have made clear in the spec, this feature is mostly required for users to debug their queries. So there's no specific need to set this at a connection level. Users should just be able to set it at a command level if they want to debug a query. But if it makes the implementation simpler to have it at a connection level, then I'm not opposed to it.

Reply:
It makes it a lot easier for `EphemeralTransaction`s, as those transactions are not exposed to the client application.